docs/Governance.md

```diff
@@ -66,8 +66,8 @@ A maintainer can also merge pull requests.
 This corresponds to the "Write" role in github.
 All maintainers should be be listed in the [CODEOWNERS file](../.github/CODEOWNERS).
 
-Personnel: @Alan-Jowett @dthaler @dv-msft @LakshK98 @matthewige @mikeagun @mtfriesen
```

Comment:
This doesn't modify the `CODEOWNERS` file - is that intentional?

Based on the criteria in this file, I probably belong in the "Contributor" role, which doesn't mention `CODEOWNERS`.

Reply:
The list of personnel in the CODEOWNERS is a superset of the list of maintainers. That is still the case after these changes. Personnel in CODEOWNERS receive email notifications for any PR.  Please reach out to me if you do not wish to be in the CODEOWNERS list.